base/module/src/fix_json.cc

```diff
@@ -52,7 +52,9 @@ struct json_converter {
     // or uses the compact format (content only, target taken from URL).
     auto add_msg_wrapper = false;
     if (is_root && v.IsObject()) {
-      if (!v.HasMember("destination") && !v.HasMember("content")) {
+      if (!(v.HasMember("destination") &&
+            v.FindMember("destination")->value.IsString()) &&
+          !v.HasMember("content")) {
```

Comment:
Müsste destination nicht in beiden Fällen ein Object sein (`Destination` oder `IntermodalDestination`)?

Vielleicht sollte man prüfen, ob `destination` *und* `content` vorhanden sind -> dann wird es wohl nicht das kompakte Format sein (`if (!(v.HasMember("destination") && v.HasMember("content")))`).